content/en/docs/components/katib/reference/experiment-cr.md

```diff
@@ -0,0 +1,57 @@
++++
+title = "Katib Experiment Lifecycle"
+description = "What happens after an `Experiment` CR is created"
+weight = 10
++++
+
+## Katib Experiment Lifecycle
+
+When user creates an Experiment, Katib Experiment controller,
+`Suggestion` controller and `Trial` controller is working together to achieve
+hyperparameters tuning for user's Machine learning model. The Experiment
+workflow looks as follows:
+
+<center>
+<img width="100%" src="/docs/components/katib/images/katib-workflow.png" alt="Katib Workflow"
+</center>
+
+1. The `Experiment` CR is submitted to the Kubernetes API server. Katib
+   `Experiment` mutating and validating webhook is called to set the default
+   values for the `Experiment` CR and validate the CR separately.
```

Comment:
Please can you make the appropriate changes in all places in this docs to rename `Experiment` CR to Experiment ?
Similar change for `Suggestion` CR and `Trial` CR.